taxes_by_invoice/templates/header_macro.html

```diff
@@ -39,8 +39,10 @@ <h1 style="font-size:20px">{% if data['parameters']['jump_page'] %}{{ _('Taxes B
             </tr>
         {% endif %}
         <tr>
-            <td>{% if data['parameters']['parties'] %}{{ _('Parties: ') }}{{ data['parameters']['parties']}}{% else %}{{ _('All Parties') }}{% endif %}</td>
-            <td style="text-align: right;">{% if data['parameters']['include_cancel'] %}{{ _('This document includes cancelled invoices marked with a *') }}{% else %}{{ _('This document not includes cancelled invoices') }}{% endif %}</td>
+            <td colspan="2">{% if data['parameters']['parties'] %}{{ _('Parties: ') }}{{ data['parameters']['parties']}}{% else %}{{ _('All Parties') }}{% endif %}</td>
+        </tr>
+        <tr>
+            <td colspan="2"><small>{{ _('Cancelled invoices are shown in grey and that those with a cancel move are not added to the total.') }}</small></td>
```

Comment:
The correct text will be:

Cancelled invoices are shown in gray and those without a cancel move or a cancel move not related to an invoice are not added to the total.

Reply:
9ea6f09c51f08d48a658527b2809a1ec12494873